### PR TITLE
Update lighting & hot water description

### PR DESCRIPTION
**Pull request by JasonEwins**
1. Updated lighting flow to pose question prior to replacing light bulb message
2. Updated 'Heating' options to 'Heating and Hot water' as per usability findings

### CODE REVIEW

File: db/questions.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ kitchen_problem:
     - text: Electrical
       next: kitchen_electrical_problem
 
-    - text: Heating
+    - text: Heating or Hot water
```
Comment (leeky) [inline db/questions.yml:96]: How about if we split these into two options, so it's easier to scan down the page?

File: db/questions.yml [hunk 2]
```diff
@@ -376,11 +376,21 @@ light_check_trip_switches:
...
+      next: light_check_light_bulb
 
     - text: 'No'
       page: check_fusebox
 
+light_check_light_bulb:
+  question: Have you replaced the light bulb?
+  answers:
+    - text: 'Yes'
+      desc: lightfitting_problem
```
Comment (leeky) [inline db/questions.yml:388]: What custom text should appear on this page?

File: db/questions.yml (hunk 1 above)
Comment (JasonEwins) [reply db/questions.yml:96]: That makes perfect sense, happy for that. 

If we can include the 'I have no heating' option within the 'heating' options as well, incase user doesn't pick up on initial emergency filter page

File: db/questions.yml (hunk 2 above)
Comment (JasonEwins) [reply db/questions.yml:388]: Content for page as per copy document - row 38 column E (with Louise's revisions)

**Comment by leeky** [comment]
Closing this in favour of a new branch which will also contain Louise's copy changes.

**Pull request closed without merging**